openreview/tools.py

```diff
@@ -1,24 +1,24 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
+from __future__ import print_function
```

Comment:
what does it do this import?

Reply:
in python 2, print is a statement:
>>> print "hello world"

in python 3, print is a function:
>>> print("hello world")

Reply:
I can change this back if you want. But there aren't really any consequences to keeping it in, and we will soon change it back anyway

Reply:
ok, if you say that this works fine, we can leave it.